conf/docker/Dockerfile

```diff
@@ -1,49 +1,85 @@
-FROM buildpack-deps:trusty
+FROM centos:7
```

Comment:
thoughts on keeping a debian based image, but updating to something like `python:3.6-buster`?

Reply:
I'll look into it, most of the dockerfile came from Josue's original Py3 PR.